draft-private-access-tokens.md

```diff
@@ -344,37 +369,26 @@ ORIGIN_NAME. An example request is shown below.
 accept = message/access-token-request
 cache-control = no-cache, no-store
 content-type = message/access-token-request
-content-length = Nk
-Token-Origin = https://example.com
-
-<Bytes containing the AccessTokenRequest>
-~~~
-
-Then the Client encapsulates this request using Oblivious HTTP, yielding an encapsulated
-HTTP message. The Client includes the "Anonymous-Origin-ID" header in this request,
-whose value is ANON_ORIGIN_ID. Finally, the Client sends this encapsulated request to the
-Mediator's proxy URI. An example request is shown below.
-
-~~~
-:method = POST
-:scheme = https
-:authority = mediator.net
-:path = /relay-access-token-request
-accept = message/ohttp-req
-cache-control = no-cache, no-store
-content-type = message/ohttp-req
-content-length = ...
+content-length = 512
+Token-Origin = ENCRYPTED_ORIGIN_NAME
 Anonymous-Origin-ID = ANON_ORIGIN_ID
 
-<Bytes containing the encapsulated HTTP message for the Issuer>
+<Bytes containing the AccessTokenRequest>
 ~~~
 
 Upon receipt, the Mediator computes ANON_CLIENT_ID, which is a fixed-length byte string
 for the given Client. See {{client-id}} for details of its computation. The Mediator
 also computes ANON_ORIGIN_ID_PRIME, a fixed-length byte string, for each ANON_ORIGIN_ID
 for a specific ANON_CLIENT_ID. See {{origin-id}} for details its computation.
 
-Before forwarding the Client's encapsulated request to the Issuer, the Mediator includes headers
+The Mediator also checks to validate that the key_id in the client's AccessTokenRequest
+matches a known public key for the Issuer. For example, the Mediator can fetch this
```

Comment:
I'm not sure if this is sufficient? Wouldn't the client be receiving keys immediately before the mediator checks the key against the issuer? (so the issuer can present key A to the client and for the next N minutes present key A to anyone who asks, before rotating to another key?)

Reply:
Added some text at the end of the paragraph to address this